### ISSUE TITLE
Release candidate in time for Scala Toolkit rollout?

### ISSUE THREAD

**Issue by SethTisue** [scalameta/munit#647]
Seems like there should be an 1.0.0-RC1 before the Toolkit is announced more broadly?

I'm not sure when @szymon-rd was thinking of launching the Scala Toolkit more formally, maybe in time for Scala Days in June...?

Has there been any past discussion of the blockers are, if any, for 1.0?

I see there is a 1.0 milestone: https://github.com/scalameta/munit/issues?q=is%3Aopen+milestone%3A%22MUnit+v1.0%22+sort%3Aupdated-desc , not sure if any of these tickets are truly blockers.

cc @adpi2

**Comment by tgodzik** [comment]
I think the idea was to finish https://github.com/scalameta/munit/milestone/1 but I don't think anyone had a time for it. We might need to work on that before the toolkit release.

**Comment by szymon-rd** [comment]
@SethTisue  We want to release 0.2 first and annouce it as a `public release`. Then, if there is some feedback or ideas on changing the libraries, we can do it for 1.0 (the `stable public release`). We are also not blocked by unstable versions of libraries, but it would be really nice to have munit 1.0-RC in 0.2.

**Comment by SethTisue** [comment]
#650 was fixed

are there any other blockers? this recently bit somebody over at https://github.com/VirtusLab/scala-cli/issues/2623

**Comment by valencik** [comment]
In my opinion, the last remaining blocker is a final decision on reverting to the v0.7.x fixture `before*`/`after*` methods.
The current issue for that is https://github.com/scalameta/munit/issues/573
And there is a recent PR to revert to v0.7.x style here: https://github.com/scalameta/munit/pull/724

Besides documentation, this is the last thing on the v1.0 milestone as well (https://github.com/scalameta/munit/milestone/1)

**Comment by valencik** [comment]
1.0.0 has been released!
https://github.com/scalameta/munit/releases/tag/v1.0.0
